Stylesheets/components/input.css

```diff
@@ -0,0 +1,61 @@
+@import url("/stylesheets/main.css");
+
+.input-holder {
+  width: fit-content;
+  margin: 1rem;
+  padding: 0.5rem;
+  background-color: var(--bg-black);
+  color: white;
+  border-radius: 0.2rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.input {
+  min-width: 20rem;
+  padding: 0.1rem;
+  border-radius: 0.1rem;
```

Comment:
for very small values, you can use pixels instead.
values such as 0.5rem and so on are okay, but 0.1rem is basically saying 1-2px. not much scope for responsiveness in such a small value.

Reply:
yes you  right but some are saying dont change units in between so i have kept everything in rem only